### ISSUE TITLE
About the training strategy

### ISSUE THREAD

**Issue by chenwc07** [mzolfaghari/ECO-efficient-video-understanding#23]
@mzolfaghari Thank you for your excellent work and the public-available model！
After reading your paper and running through the code(ECO-pytorch), I have some question about the training strategy. I followed the training parameter in the training script to finetune the ECO-lite model on UCF101 using 16 frames, but I failed to get the same result as in the paper which is 91.6% on the 3 splits of UCF101. 
Then I tried to finetune the ECO-full model using the same strategy, but I got the nearly same result as ECO-lite, which is approximately 88%. It seems that ECO-full does not improve the performance. So I wonder if there is any difference between the training parameters of lite and full model. 
Thank you!

**Comment by mzolfaghari** [comment]
@chenwc07 Seems that the same hyperparameters can not exactly reproduce the results we obtained using Caffe. Using PyTorch, on the UCF101 dataset split1 we got 91.7% accuracy using the following hyperparameters:
dropout: 0.6
learning rate: 0.001
batch size: 50
We initialized the weights with pre-trained model on Kinetics and trained the model for 50 epochs.
Unfortunately, we don't have the results for the full version. We will provide all models and results using PyTorch version after the CVPR deadline.

* For the ECO Full version, you can initialize the 3D part of the network with ECO-lite trained on Kinetics and the 2D part of the network with TSN network trained on Kinetics. If you carefully initialize the weights then you probably will be able to reproduce the results that we reported for ECO Full version.

**Comment by chenwc07** [comment]
@mzolfaghari Thank you for your reply! I will try these hyperparameters.

**Comment by sophia-wright-blue** [comment]
@chenwc07 were you able to reproduce the results with the new hyperparameters? thanks,

**Comment by chenwc07** [comment]
@sophia-wright-blue I got 91.4% accuracy on the UCF101 dataset split1 using 16F lite model
